### PR TITLE
Fix php warnings.

### PR DESCRIPTION
**Pull request by YevhenShBolt**
These fixes prevent fall the order saving process.
Especially I caught these errors in admin order creation process in one merchant.

The instance based on php5.6 and Magento1.9.4.x

### CODE REVIEW

File: app/code/community/Bolt/Boltpay/Model/Admin/ExtraConfig.php [hunk 1]
```diff
@@ -181,16 +181,16 @@ public function getExtraConfig($configName, $filterParameters = array() ) {
...
+     * @param string $string    The string to be stripped of newlines
      *
-     * @return string|null    The string stripped of newline characters or null on error
+     * @return string|null      The string stripped of newline characters or null on error
      */
-    private function normalizeJSON($string) {
-        return trim(preg_replace( '/(\r\n)|\n|\r/', '', $string )) ?: array();
+    private function normalizeJSON($string)
+    {
+        return trim(preg_replace( '/(\r\n)|\n|\r/', '', $string )) ?: '';
```
Comment (mccottry) [inline app/code/community/Bolt/Boltpay/Model/Admin/ExtraConfig.php]: This change to returning an array is a very important and very intentional change that was made in version 1.4.0.  Please review Ning's post that is pinned to the top of this thread regarding errors thrown by json_decode on empty stings in PHP 7.0.0

https://app.asana.com/0/inbox/544708310157128/1110257233649801/1110933955640367

Comment (YevhenShBolt) [reply app/code/community/Bolt/Boltpay/Model/Admin/ExtraConfig.php]: In one merchant in php5.6 we have a problem with this warning

Comment (dumega) [reply app/code/community/Bolt/Boltpay/Model/Admin/ExtraConfig.php]: @YevhenShBolt 

I just test your PR on my dev site, the error `Invalid JSON for Bolt Extra Options. Syntax error` is back again, and the PHP version is v7.1.27. Then I suggest to change the empty string back to an empty array.

image

**Comment by YevhenShBolt** [comment]
I tested on php5.6 and 7.0 and 7.1. The error do not appear any more.
Now returns the proper json value for `json_decode` command.

**Pull request merged**